advisories/github-reviewed/2024/12/GHSA-mfj5-cf8g-g2fv/GHSA-mfj5-cf8g-g2fv.json

```diff
@@ -25,16 +25,13 @@
           "type": "ECOSYSTEM",
           "events": [
             {
-              "introduced": "3.0.0"
+              "introduced": "2.0.0"
```

Comment:
```suggestion
              "introduced": "2.1.0"
```

Reply:
I had tried to apply this myself without realizing I don't have permission based on

https://github.com/AsyncHttpClient/async-http-client/blob/main/CHANGES.md#from-20-to-21

Reply:
All good. I'll take care of the change on the backend. Many thanks 👍